packages/android-client/FishjamClient/src/main/java/com/fishjamcloud/client/events/Event.kt

```diff
@@ -1,429 +1,15 @@
-  @SerializedName("connected")
-  Connected,
-
-  @SerializedName("endpointAdded")
-  EndpointAdded,
-
-  @SerializedName("endpointUpdated")
-  EndpointUpdated,
-
-  @SerializedName("endpointRemoved")
-  EndpointRemoved,
-
-  @SerializedName("custom")
-  Custom,
-
-  @SerializedName("offerData")
-  OfferData,
-
-  @SerializedName("candidate")
-  Candidate,
-
-  @SerializedName("tracksAdded")
-  TracksAdded,
-
-  @SerializedName("tracksRemoved")
-  TracksRemoved,
-
-  @SerializedName("trackUpdated")
-  TrackUpdated,
-
-  @SerializedName("sdpAnswer")
-  SdpAnswer,
-
-  @SerializedName("encodingSwitched")
-  EncodingSwitched,
-
-  @SerializedName("vadNotification")
-  VadNotification,
-
-  @SerializedName("bandwidthEstimation")
-  BandwidthEstimation
-}
-
-internal data class BaseReceivableEvent(
-  val type: ReceivableEventType
-)
-
-sealed class ReceivableEvent {
-  companion object {
-    fun decode(payload: Payload): ReceivableEvent? {
-      try {
-        val eventBase: BaseReceivableEvent = payload.toDataClass()
-
-        return when (eventBase.type) {
-          ReceivableEventType.Connected ->
-            payload.toDataClass<Connected>()
-
-          ReceivableEventType.EndpointAdded ->
-            payload.toDataClass<EndpointAdded>()
-
-          ReceivableEventType.EndpointRemoved ->
-            payload.toDataClass<EndpointRemoved>()
-
-          ReceivableEventType.EndpointUpdated ->
-            payload.toDataClass<EndpointUpdated>()
-
-          ReceivableEventType.TracksAdded ->
-            payload.toDataClass<TracksAdded>()
-
-          ReceivableEventType.TracksRemoved ->
-            payload.toDataClass<TracksRemoved>()
-
-          ReceivableEventType.TrackUpdated ->
-            payload.toDataClass<TrackUpdated>()
-
-          ReceivableEventType.Custom -> {
-            val customEventBase = payload.toDataClass<BaseCustomEvent>()
-
-            return when (customEventBase.data.type) {
-              ReceivableEventType.OfferData ->
-                payload.toDataClass<CustomEvent<OfferData>>().data
-
-              ReceivableEventType.Candidate ->
-                payload.toDataClass<CustomEvent<RemoteCandidate>>().data
-
-              ReceivableEventType.SdpAnswer ->
-                payload.toDataClass<CustomEvent<SdpAnswer>>().data
-
-              ReceivableEventType.EncodingSwitched ->
-                payload.toDataClass<CustomEvent<EncodingSwitched>>().data
-
-              ReceivableEventType.VadNotification ->
-                payload.toDataClass<CustomEvent<VadNotification>>().data
-
-              ReceivableEventType.BandwidthEstimation ->
-                payload.toDataClass<CustomEvent<BandwidthEstimation>>().data
-
-              else ->
-                null
-            }
-          }
-
-          else ->
-            null
-        }
-      } catch (e: JsonParseException) {
-        Timber.e(e)
-        return null
-      }
-    }
+internal fun String.serializeToMap(): Map<String, Any?> =
```

Comment:
This file only has this one function and it's not an event so maybe we should consider changing the filename?

Reply:
Good point, it used to have Events but now they are gone. I forgot to rename the file.